akka-docs/src/main/paradox/cluster-usage.md

```diff
@@ -12,6 +12,54 @@ To use Akka Cluster, you must add the following dependency in your project:
   version="$akka.version$"
 }
 
+## When and where to use Akka Cluster
+
+An architectural choice you have to make is if you are going to use a microservices architecture or
+a traditional distributed application, also known as a distributed monolith. This choice will influence how
```

Comment:
That's what I suggested. That:
1) We simply call it a "traditional distributed system", since it is a traditional/classic distributed system architected the way we have built these systems for decades, and explain its benefit.
2) Reference that this architecture is sometimes referred to as a "distributed monolith anti-pattern" in the microservices community, and clearly explain why it is called that in this context, by discussing its problems with coupling etc. Make it clear that a user has to choose between this pattern and microservices.

Reply:
I agree, let's not use the word here. It's only adding confusion. I think the anti-pattern is mostly when you think you are building microservices but they are so tightly coupled (with RPC, dependencies or shared DB) that they effectively are a monolith. The distinction is if it was a conscious decision or not.